packages/bdd-tests/util/helper.ts

```diff
@@ -20,25 +20,25 @@ export let longPrescId = ""
 let identifierValue = ""
```

Comment:
We shouldn't have vars at this scope, at the top of the file like this. It's only string constants that tend to go here.